tools/debug-ui/src/api.tsx

```diff
@@ -435,10 +435,22 @@ export async function fetchTrackedShards(addr: string): Promise<TrackedShardsRes
 
 export async function fetchBlockStatus(
     addr: string,
-    height: number | null
+    height: number | null,
+    mode: string | null,
+    numBlocks: number | null
 ): Promise<BlockStatusResponse> {
-    const trailing = height ? `/${height}` : '';
-    const response = await fetch(`http://${addr}/debug/api/block_status${trailing}`);
+    const params = new URLSearchParams();
+    if (height !== null) {
+        params.append('starting_height', height.toString());
+    }
+    if (mode !== null) {
+        params.append('mode', mode);
+    }
+    if (numBlocks !== null) {
+        params.append('num_blocks', numBlocks.toString());
+    }
+    const url = `http://${addr}/debug/api/block_status${params.toString() ? '?' + params : ''}`;
```

Comment:
Since old neard versions expect `/debug/api/block_status/{starting_height}`, now the block height text field won't work with a new version of the debug UI but old neard. But I agree that putting the height in the URL params makes more sense, so this breaking change is prob not a huge deal.  Idk how worth it it is, but if you want, could consider keeping the old url format with the new params until the old version is prob no longer running anywhere

Reply:
I don't actually break connection from new UI to old neard: https://github.com/near/nearcore/pull/12930/files#diff-1a387e5429d179d79bfac928c9a71fc733feeeef8033332f4878d58c798d5f44R1665-R1674

The `deprecated_debug_block_status_handler` is still here, and it should stay until all `neard`s are updated. By default, it will be used for all addresses matching `/debug/api/block_status/{starting_height}`. Calls from new UI will use URL params and won't go there.